packages/repository-json-schema/src/__tests__/integration/build-schema.integration.ts

```diff
@@ -179,6 +180,23 @@ describe('build-schema', () => {
         expectValidJsonSchema(jsonSchema);
       });
 
+      it('properly converts any properties', () => {
+        @model()
+        class TestModel {
+          @property({
+            type: 'any',
+          })
+          anyProp: AnyType;
+        }
+
+        const jsonSchema = modelToJsonSchema(TestModel);
+        expect(jsonSchema.properties).to.deepEqual({
+          anyProp: {},
```

Comment:
Do we prefer `{}` or `true`?

Reply:
@raymondfeng The reason I eventually chose `{}` is because the object can take in other properties like `description`, see [code](https://github.com/strongloop/loopback-next/blob/master/packages/repository-json-schema/src/build-schema.ts#L246-L254)
I verified `{description: 'some desc'}` is still a valid syntax for any type.
WDYT? I can switch to `true` if we want to align with the latest standard.

Reply:
As long as we test `true`, I'm good either way.

Reply:
See https://json-schema.org/draft/2019-09/json-schema-core.html#rfc.section.4.3.2

Reply:
sounds good, I added another 2 set for `true`.